resources/views/emails/visit/new.blade.php

```diff
@@ -3,10 +3,14 @@
 @section('content')
 <p>Dear {{ $visit->name }},</p>
 
-<p>Your visitor application has been successfully submitted. In the next 24 hours, you will be assigned two exams via the VATUSA Exam Center. The first part of the GRP check will ensure you have the skills required to control at ZTL. Some questions will require you to reference various SOPs and LOAs that can be found on the <a href="www.ztlartcc.org/dashboard/controllers/files">ZTL files page.</a></p>
-<p>The second part of the GRP check will ensure you understand the rules for being a member at ZTL. The answers to these questions can be found in our Facility Administrative Policy on the files page.</p>
-<p>You will have seven (7) days to complete the exams and you must pass each with a score of 80% or better. If you have any questions or concerns, please email the DATM at <a href="mailto:datm@ztlartcc.org">datm@ztlartcc.org</a>.</p>
-
+<p>Your visitor application has been successfully submitted. In the next 24 hours, you will be assigned the ZTL Visitor Exam through the 
+    VATUSA Academy. Please visit the <a href="https://academy.vatusa.net/">VATUSA Academy</a> and under "Academy Courses" select "All 
+    Courses". You should then see the ZTL Visiting Exams course once it has been assigned.</p>
```

Comment:
Small grammar error: technically it should be. Approving - your call on this change (nobody will notice it).

```diff
- "All Courses".
+ "All Courses."
```